src/Quartz/SchedulerBuilder.cs

```diff
@@ -592,20 +592,54 @@ public static class AdoProviderExtensions
             configurer.Invoke(adoProviderOptions);
         }
 
+        /// <summary>
+        /// Configures the scheduler to use System.Data.Sqlite data source provider.
+        /// </summary>        
         public static void UseSQLite(
             this SchedulerBuilder.PersistentStoreOptions options,
             string connectionString)
         {
             options.UseSQLite(c => c.ConnectionString = connectionString);
         }
 
+        /// <summary>
+        /// Configures the scheduler to use System.Data.Sqlite data source provider.
+        /// </summary>
         public static void UseSQLite(
             this SchedulerBuilder.PersistentStoreOptions options,
             Action<SchedulerBuilder.AdoProviderOptions> configurer)
+        {
+            options.UseSQLite("SQLite", configurer);
+        }
+
+        /// <summary>
+        /// Configures the scheduler to use Microsoft.Data.Sqlite data source provider.
+        /// </summary>        
+        public static void UseMSSQLite(
```

Comment:
Would it make sense to call these `UseMicrosoftSQLite`, this abbreviation is quite the monster 😄

Reply:
Fixed